### PR TITLE
Print a warning for repos without descriptions

### PR DESCRIPTION
**Pull request by jepler**
These descriptions improve social shares, and possibly search results.

Missing repository description - 60
  * https://github.com/adafruit/Adafruit_CircuitPython_SK9822
  * https://github.com/adafruit/Adafruit_CircuitPython_Debug_Bus_Device
  * https://github.com/adafruit/Adafruit_CircuitPython_Debug_SPI
  * https://github.com/adafruit/Adafruit_CircuitPython_74HC595
  * https://github.com/adafruit/Adafruit_CircuitPython_ADXL34x
  * https://github.com/adafruit/Adafruit_CircuitPython_APDS9500
  * https://github.com/adafruit/Adafruit_CircuitPython_BD3491FS
  * https://github.com/adafruit/Adafruit_CircuitPython_BH1750
  * https://github.com/adafruit/Adafruit_CircuitPython_BluefruitConnect
  * https://github.com/adafruit/Adafruit_CircuitPython_BMP3XX
  * https://github.com/adafruit/Adafruit_CircuitPython_Debug_I2C
  * https://github.com/adafruit/Adafruit_CircuitPython_DS1841
  * https://github.com/adafruit/Adafruit_CircuitPython_DS3502
  * https://github.com/adafruit/Adafruit_CircuitPython_ESP_ATcontrol
  * https://github.com/adafruit/Adafruit_CircuitPython_FRAM
  * https://github.com/adafruit/Adafruit_CircuitPython_framebuf
  * https://github.com/adafruit/Adafruit_CircuitPython_HTS221
  * https://github.com/adafruit/Adafruit_CircuitPython_HX8357
  * https://github.com/adafruit/Adafruit_CircuitPython_ICM20X
  * https://github.com/adafruit/Adafruit_CircuitPython_INA260
  * https://github.com/adafruit/Adafruit_CircuitPython_IterTools
  * https://github.com/adafruit/Adafruit_CircuitPython_LIS2MDL
  * https://github.com/adafruit/Adafruit_CircuitPython_LIS331
  * https://github.com/adafruit/Adafruit_CircuitPython_LIS3MDL
  * https://github.com/adafruit/Adafruit_CircuitPython_LPS2X
  * https://github.com/adafruit/Adafruit_CircuitPython_LPS35HW
  * https://github.com/adafruit/Adafruit_CircuitPython_LSM303_Accel
  * https://github.com/adafruit/Adafruit_CircuitPython_LSM303DLH_Mag
  * https://github.com/adafruit/Adafruit_CircuitPython_MAX31856
  * https://github.com/adafruit/Adafruit_CircuitPython_MCP4728
  * https://github.com/adafruit/Adafruit_CircuitPython_MIDI
  * https://github.com/adafruit/Adafruit_CircuitPython_miniesptool
  * https://github.com/adafruit/Adafruit_CircuitPython_MLX90614
  * https://github.com/adafruit/Adafruit_CircuitPython_MPL115A2
  * https://github.com/adafruit/Adafruit_CircuitPython_MSA301
  * https://github.com/adafruit/Adafruit_CircuitPython_PCF8591
  * https://github.com/adafruit/Adafruit_CircuitPython_PyBadger
  * https://github.com/adafruit/Adafruit_CircuitPython_RockBlock
  * https://github.com/adafruit/Adafruit_CircuitPython_RPLIDAR
  * https://github.com/adafruit/Adafruit_CircuitPython_SHTC3
  * https://github.com/adafruit/Adafruit_CircuitPython_SSD1305
  * https://github.com/adafruit/Adafruit_CircuitPython_SSD1331
  * https://github.com/adafruit/Adafruit_CircuitPython_SSD1351
  * https://github.com/adafruit/Adafruit_CircuitPython_TLV493D
  * https://github.com/adafruit/Adafruit_CircuitPython_TMP006
  * https://github.com/adafruit/Adafruit_CircuitPython_TPA2016
  * https://github.com/adafruit/Adafruit_CircuitPython_VCNL4040
  * https://github.com/adafruit/Adafruit_CircuitPython_VEML7700
  * https://github.com/adafruit/Adafruit_CircuitPython_RA8875
  * https://github.com/adafruit/Adafruit_CircuitPython_GFX
  * https://github.com/adafruit/Adafruit_CircuitPython_PN532
  * https://github.com/adafruit/Adafruit_CircuitPython_CLUE
  * https://github.com/adafruit/Adafruit_CircuitPython_DPS310
  * https://github.com/adafruit/Adafruit_CircuitPython_BitbangIO
  * https://github.com/adafruit/Adafruit_CircuitPython_MPU6050
  * https://github.com/adafruit/Adafruit_CircuitPython_Pypixelbuf
  * https://github.com/adafruit/Adafruit_CircuitPython_AS7341
  * https://github.com/adafruit/Adafruit_CircuitPython_LSM6DS
  * https://github.com/adafruit/Adafruit_CircuitPython_PCT2075
  * https://github.com/adafruit/Adafruit_CircuitPython_MS8607

### CODE REVIEW

File: adabot/circuitpython_libraries.py [hunk 1]
```diff
@@ -230,13 +230,14 @@ def run_library_checks(validators, bundle_submodules, latest_pylint, kw_args):
...
-            len(lib_insights["open_prs"]),
-            max(open_pr_days),
-            max((min(open_pr_days), 1)) # ensure the minumum is '1'
+    if len(lib_insights["open_prs"]) != 0:
+        output_handler(
+            "  * {0} open pull requests (Oldest: {1}, Newest: {2})".format(
+                len(lib_insights["open_prs"]),
+                max(open_pr_days),
+                max((min(open_pr_days), 1)) # ensure the minumum is '1'
+            )
```
Comment (kattni) [inline adabot/circuitpython_libraries.py:239]: Why was this change necessary?

Comment (sommersoft) [reply adabot/circuitpython_libraries.py:239]: I _think_ its to guard against an exception with `max(open_pr_days)`, when `open_pr_days` is an empty list (zero open PRs). Not a bad guard to have, imo.

Comment (jepler) [reply adabot/circuitpython_libraries.py:239]: This was noted in the individual commit message, but I didn't call it out again in the PR summary:
>  circuitpython_libraries.py: Don't error if open_prs is empty
>
> This could occur during my testing, which I restricted manually to a tiny subset of libs.  Otherwise, it is unlikely to occur.
